### PR TITLE
Update AboutUs, User Guide and Developer Guide for v1.1

### PR DESCRIPTION
**Pull request by dt-td**

### CODE REVIEW

File: docs/team/dt-td.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (wpinrui) [inline docs/team/dt-td.md:1]: LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -283,33 +283,105 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `TutorAid` and the **Actor** is the `tutor`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case 5: Updating payment for student**
```
Comment (wpinrui) [inline docs/DeveloperGuide.md:288]: LGTM

File: docs/UserGuide.md [hunk 3]
```diff
@@ -142,6 +142,20 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Viewing a student : `view`
```
Comment (wpinrui) [inline docs/UserGuide.md:145]: LGTM

File: src/main/resources/view/MainWindow.fxml [hunk 4]
```diff
@@ -12,7 +12,7 @@
 <?import javafx.scene.layout.VBox?>
 
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1"
-         title="Address App" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="TutorAid" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
```
Comment (wpinrui) [inline src/main/resources/view/MainWindow.fxml:15]: Should we do this in a separate commit?

Comment (Preshita01) [reply src/main/resources/view/MainWindow.fxml:15]: I agree with this as well, might make more sense to make this change is a future commit since it is editing the MainWindow?

File: docs/AboutUs.md [hunk 5]
```diff
@@ -29,3 +29,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Documentation
 
+### Dickson
```
Comment (Preshita01) [inline docs/AboutUs.md:42]: LGTM

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -283,32 +283,104 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-*{More to be added}*
+* 3b. The given student index is invalid
+ 
+    * 3b1. TutorAid shows an error message
+    * 3b2. User enters a new index with paid/unpaid command
+  
+      Steps 3a1 - 3a2 are repeated until a valid index is given
+      Use case resumes from step 4
+    
+* 4a. Incorrect update of payment for a different student instead
```
Comment (Preshita01) [inline docs/DeveloperGuide.md:402]: I was wondering if extension 4a is needed? This is because extensions usually describe exceptional/alternative flow of events, where there is a disruption when going through the steps stated in the MSS. But for 4a the tutor will technically be able to complete the whole MSS although the payment update is being made for the wrong student?

Otherwise LGTM

**Review by Preshita01 — APPROVED** [review]
LGTM other than the small suggestions

File: src/main/resources/view/MainWindow.fxml (hunk 4 above)
Comment (dt-td) [reply src/main/resources/view/MainWindow.fxml:15]: Okay will revert this back. Thank you!
